ui/oopsyraidsy/oopsy_options.ts

```diff
@@ -47,7 +47,7 @@ const defaultOopsyNonConfigOptions: OopsyNonConfigOptions = {
 export type DeathReportSide = 'left' | 'right' | 'disabled';
 
 type OopsyConfigOptions = {
-  Debug: boolean;
+  Debug?: boolean;
```

Comment:
Can you make this non-optional, here and elsewhere? And then conditionals can just be `if (thing)` and not `if (thing === true)`.